### ISSUE TITLE
IAST glyphs

### ISSUE THREAD

**Issue by chaw** [rosettatype/eczar#19]
This is a feature request, not a bug report.

It would be nice to have glyphs for the characters used by the IAST scheme,  such as the Latin letters with dots under them, e.g.,  ḍ, ṇ, ṛ, ṣ, etc.  I believe the relevant Unicode block is Latin Extended Additional, 1E00 through 1EFF, but only a few of those are needed for IAST.

Having those glyphs would permit the use of Eczar in combined Sanskrit-IAST-English documents that are common in some scholarly work, for instance.  (If I'm not mistaken, this font has some origins in multi-script typesetting.)  Of course, those glyphs can be produced in other ways by a typesetting system, but I think having the glyphs included in the font itself would be nicer.

In any case, thanks for all the work on this very nice  font!

**Comment by MrBrezina** [comment]
@chaw I agree. Perhaps @errorsinc will add them one day. In the mean time, if need be, you can use Skolar or Skolar Sans. Those are commercial, though. http://rosettatype.com/Skolar and http://rosettatype.com/SkolarSans

btw. nothing prevents you from forking the repo and adding them yourself. They are easy to add as most of them are composites of existing glyphs.

**Comment by chaw** [comment]
Thanks for the quick response and suggestions.  I will look into adding some glyphs.
